site/content/en/docs/manual/advanced/serverless-tutorial.md

```diff
@@ -898,6 +898,59 @@ $ docker logs nuclio-nuclio-pth.shiyinzhang.iog
 
 </details>
 
+### Problem with running nuctl on Windows 10
+
+If you encounter the problem with running `nuctl` on Windows 10, you can do the following:
+
+1. Install [Ubuntu 18.04 from Microsoft store][ubuntu-18-04-microsoft-store].
+
+1. Enable integration for Ubuntu-18.04 in the settings `Docker Desktop` in the `Resourcrces` `WSL integration` tab
+   (it is understood that `Docker` and `WSL 2` are installed on windows):
+
+   ![Docker WSL integration Ubuntu 18.04](/images/docker_wsl_integration.jpg)
+
+1. Install `curl` on Ubuntu with use command:
+   ```sh
+   sudo apt install curl
+   ```
+1. Installation `nuctl` on Ubuntu, using the command from [manuals on nuclio CLI][nuctl-manual]:
+   ```sh
+   curl -s https://api.github.com/repos/nuclio/nuclio/releases/1.5.16 \
+               | grep -i "browser_download_url.*nuctl.*$(uname)" \
+               | cut -d : -f 2,3 \
+               | tr -d \" \
+               | wget -O nuctl -qi - && chmod +x nuctl
+   ```
+1. Installation `git` on Ubuntu:
+   ```sh
+   sudo apt install git
+   ```
+1. Clone the repository and move to the repository folder:
+   ```sh
+   git clone https://github.com/openvinotoolkit/cvat.git
+   cd cvat
+   ```
+
+1. After that, run the commands from this tutorial through Ubuntu.
+
+### Problem with the deployment of models
```

Comment:
I would add the section into troubleshooting guide. If the user sees that the port is 0, need to assign it manually.